astropy/modeling/tests/test_quantities_evaluation.py

```diff
@@ -317,6 +318,40 @@ def test_compound_input_units_equivalencies():
     assert exc.value.args[0] == "Shift: Units of input 'x', pix (unknown), could not be converted to required input units of deg (angle)"
 
 
+def test_compound_model_input_units_equivalencies_defaults():
+    """
+    Test input_units_equivalencies default value for compound models
+    """
+
+    test_models = []
+    test_models.append(Shift(10 * u.deg))
+    test_models.append(Polynomial1D(1, c0=1, c1=2))
+    test_models.append(Scale(2))
+
+    for test_model in test_models:
+        assert test_model.input_units_equivalencies == None
+
+        for op in ['&','+', '|']:
+
+            if op == '+':
+                test_compound_model = test_model + test_model
+                assert test_compound_model.inputs_map()['x'][0].input_units_equivalencies == None
+                fixed_input_model = fix_inputs(test_compound_model, {'x':1})
+
+            elif op == '&':
+                test_compound_model = test_model & test_model
+                assert test_compound_model.inputs_map()['x1'][0].input_units_equivalencies == None
+                fixed_input_model = fix_inputs(test_compound_model, {'x0':1})
+                assert fixed_input_model.inputs_map()['x1'][0].input_units_equivalencies == None
+
+            elif op == '|':
+                test_compound_model = test_model | test_model
+                assert test_compound_model.inputs_map()['x'][0].input_units_equivalencies == None
+                fixed_input_model = fix_inputs(test_compound_model, {'x':1})
+
+            assert fixed_input_model.input_units_equivalencies == None
+
+
```

Comment:
Please write this test as a parametrized pytest test. Here's an [example](https://github.com/astropy/astropy/blob/1b8b4d17864223678003e8867852c19369ea28ce/astropy/modeling/tests/test_models_quantities.py#L213). 

```
@pytest.mark.parametrize('model', [Shift(10 * u.deg), Polynomial1D(1, c0=1, c1=2), Scale(2)])
def test_compound_model_input_units_equivalencies_defaults(model):
```
This will pass one model at a time.
Don't loop over operators, simply write the expression for the compound model using each operator.

Reply:
Done.

After seeing the models in `test_models_quantities.py` I decided the move the test their and use the same models as parameters. I hope it still makes sense for the test to be there